Dockerfile

```diff
@@ -24,7 +35,7 @@ COPY go.sum .
 
 RUN go mod download
 
-RUN make build
+RUN cd simapp && GOOS=linux GOARCH=amd64 go build -mod=readonly -tags "netgo ledger muslc" -ldflags '-X github.com/cosmos/cosmos-sdk/version.Name=sim -X github.com/cosmos/cosmos-sdk/version.AppName=simd -X github.com/cosmos/cosmos-sdk/version.Version= -X github.com/cosmos/cosmos-sdk/version.Commit= -X "github.com/cosmos/cosmos-sdk/version.BuildTags=netgo ledger muslc," -w -s -linkmode=external -extldflags "-Wl,-z,muldefs -static"' -trimpath -o /go/build/ ./...
```

Comment:
It looks like `-linkmode=external -extldflags "-Wl,-z,muldefs -static"` is not usually included when we run `make build`. Should I try to include these changes there as well? Folks okay if I hardcode it for now?

Reply:
did a quick peek into changing this, but ran into a little difficulties, so I'll let someone be my guest if they want to adjust it